dist/main.css

```diff
@@ -293,6 +293,12 @@
     .govuk-list--number > li {
       margin-bottom: 5px; } }
 
+.govuk-list--spaced > li {
```

Comment:
Could you add the command used here? (Or perhaps it should go in the README.)

Reply:
By running `npm run build` which is also triggered before the test suite is run.